uk_bin_collection/uk_bin_collection/councils/TamworthBoroughCouncil.py

```
@@ -0,0 +1,67 @@
+
+        headers = {
+            "Origin": "https://www.lichfielddc.gov.uk",
+            "Referer": "https://www.lichfielddc.gov.uk",
+            "User-Agent": "Mozilla/5.0",
+        }
+
+        # Tamworth waste services are operated by Lichfield District Council
+        URI = f"https://www.lichfielddc.gov.uk/homepage/6/bin-collection-dates?uprn={user_uprn}"
+
+        # Make the GET request
+        response = requests.get(URI, headers=headers)
```

Ruff (0.15.15)

[error] Probable use of `requests` call without timeout

(S113)